wrapanapi/systems/msazure.py

```diff
@@ -1130,6 +1166,21 @@ def delete_stack_by_date(self, days_old, resource_group=None):
                              result)
         return results
 
+    def delete_compute_image_using_name(self, resource_group=None, image_list=None):
```

Comment:
Instead this should be delete images same as you have in EC2 systems. We cannot say by name becuase we are not passing as parameter name , we are just passing image_list be its all free images list or list of all images !

Reply:
Changed the name to delete_compute_image_by_resource_group()